tests/test_statespace/test_discrete_transition.py

```diff
@@ -369,3 +368,119 @@ def test_process_noise_cov_cholesky(self):
         received = self.transition.proc_noise_cov_cholesky
         expected = np.linalg.cholesky(self.S_const)
         np.testing.assert_allclose(received, expected)
+
+
+class TestLinearGaussianLinOps:
+    """Test class for linear Gaussians using LinearOperators where possible.
+
+    Also tests that different forward and backward deal correctly when LinearOperators
+    are used.
+    """
+
```

Comment:
stupid (perhaps blunt) question: does this test suite is a C&P from the LinearGaussian testcase, just with linops everywhere? Do you think there is a slimmer version of running these tests (in terms of additional parameters in the other test case). If so, that might be a good template of "how to do this kind of linop refactoring in statespace models".

Reply:
There might be a slimmer version, but it is not straightforward, imo. And in the end some tests might be ran that make no sense. So I figured, this is the easier solution.

Reply:
Hmm okay thanks for the assessment :) I think it is okay, but was just wondering.